README.md

```diff
@@ -17,7 +17,113 @@ TerseTS can be compiled and cross-compiled from source:
    - Microsoft Windows: `zig build -Dtarget=x86_64-windows -Doptimize=ReleaseFast`
 
 # Usage
-TODO
+
+To use TerseTS in a C project, follow these steps:
```

Comment:
I excluded Objective-C as I did not know how to create an example for it. Regardless, maybe it is implicit now that C/C++ is included.

Reply:
In [contrast to C++](https://en.wikipedia.org/wiki/C%2B%2B#With_C), Objective-C is a [super set of C](https://en.wikipedia.org/wiki/Objective-C#Syntax). So an example written in C can also be used with Objective-C, it just cannot make use of the Smalltalk-style object-oriented features added by Objective-C. However, since the example is written in C and not C++ or makes use of the extensions added by Objective-C, maybe just write `C Usage Example`. This make it clear to C++ and Objective-C programmers that there is a C-API but not a C++ or Objective-C API.

Reply:
Interesting. Fixed.